src/api/include/migraphx/migraphx.hpp

```diff
@@ -66,7 +66,7 @@ template <class PrivateMigraphTypeNameProbe>
 std::string compute_type_name()
 {
     std::string name;
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
why does it requires check for clang. Aren't both exclusive if it is  MSVC, it can't be clang.

Reply:
The Clang compiler on Windows is running in MSVC compatibility mode. It sets `_MSC_VER`, too. So, only checking `_MSC_VER` is not exclusive and is insufficient to differentiate between native MSVC and Clang. See detailed information https://clang.llvm.org/docs/MSVCCompatibility.html